plugins/ocp/ocp-telemetry-decode.c

```diff
@@ -642,43 +644,60 @@ void parse_time_stamp_event(struct nvme_ocp_telemetry_event_descriptor *pevent_d
 {
 	struct nvme_ocp_time_stamp_dbg_evt_class_format *ptime_stamp_event =
 		(struct nvme_ocp_time_stamp_dbg_evt_class_format *) pevent_specific_data;
-
-	int vu_event_id = (int)ptime_stamp_event->vu_event_identifier;
-
-	unsigned int data_size = ((pevent_descriptor->event_data_size * SIZE_OF_DWORD)-
-					sizeof(struct nvme_ocp_time_stamp_dbg_evt_class_format));
-
-	__u8 *pdata = (__u8 *)ptime_stamp_event +
-					sizeof(struct nvme_ocp_time_stamp_dbg_evt_class_format);
-
+	struct nvme_ocp_common_dbg_evt_class_vu_data *ptime_stamp_event_vu_data = NULL;
+	__le16 vu_event_id = 0;
+	__u8 *pdata = NULL;
 	char description_str[256] = "";
-
-	parse_ocp_telemetry_string_log(0, ptime_stamp_event->vu_event_identifier,
-				       pevent_descriptor->debug_event_class_type,
-				       VU_EVENT_STRING, description_str);
+	unsigned int vu_data_size = 0;
+	bool vu_data = false;
```

Comment:
More of a nit, but I'd assume this was the actual vendor unique data. Maybe `vu_data_present` would be clearer?

As above, any change here should also be made in the other event class functions like PCIe Debug, etc.

Reply:
Changed vu_data to vu_data_present.